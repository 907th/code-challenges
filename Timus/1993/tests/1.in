(We) [are] {blind}, if (we) [could not see] {creation of this clone army}
