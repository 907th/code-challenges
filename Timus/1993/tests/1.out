Blind we are, if creation of this clone army we could not see
